web/ui/react-app/src/pages/targets/SeriesLabels.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from 'react';
+import { Tooltip } from 'reactstrap';
+import SeriesLabel from './SeriesLabel';
+import styles from './SeriesLabels.module.css';
+
+interface Labels {
+  [key: string]: string;
+}
+
+export interface SeriesLabelsProps {
+  discoveredLabels: Labels;
+  labels: Labels;
+  idx: number;
+  scrapePool: string;
+}
+
+const formatLabels = (labels: Labels): string[] => Object.keys(labels).map(key => `${key}="${labels[key]}"`);
+
+const SeriesLabels: FC<SeriesLabelsProps> = ({ discoveredLabels, labels, idx, scrapePool }: SeriesLabelsProps) => {
```

Comment:
Thinking about this, `<SeriesLabel>` and `<SeriesLabels>` is a misnomer, since it's not about series yet, but about targets (what we call target labels). This component is even coded specifically around targets in that it has a tooltip with labels from before relabeling, which only applies to targets, not series. Can we rename it `<TargetLabel>` / `<TargetLabels>`?

Reply:
Yes will do!

Reply:
done